x-pack/plugins/reporting/export_types/printable_pdf/server/lib/layouts/layout.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { kbn_server } from '../../../../../../../../src/server/index';
+
+export class Layout {
+  public id: string = '';
+  public server: kbn_server;
```

Comment:
I'd get rid of this variable on the base class and only put it on the Print layout class. I think it's only used there, for grabbing the captureConfig.